.travis.yml

```diff
@@ -10,15 +10,14 @@ script:
   - ng e2e --protractor-config=e2e/protractor-ci.conf.js --prod
   - npm run build-prod:ssr
   - cp dist/frontend/browser/index.html dist/frontend/browser/404.html
+  - docker build -t avrae-io .
 deploy:
-  provider: pages
-  local_dir: dist/frontend/browser
-  skip_cleanup: true
-  github_token: $GITHUB_TOKEN
-  keep_history: true
-  fqdn: avrae.io
+  provider: script
+  script: bash scripts/ecr_push.sh
```

Comment:
It MIGHT be worth moving this over to codebuild, but your call.

Reply:
I talked to Aaron about that when I started this. We decided it wasn't worth the hassle.